ksql-engine/src/main/java/io/confluent/ksql/util/QueryIdGenerator.java

```diff
@@ -14,19 +14,16 @@
 
 package io.confluent.ksql.util;
 
-import java.util.Objects;
 import java.util.concurrent.atomic.AtomicLong;
 
 public class QueryIdGenerator {
 
   private final AtomicLong queryIdCounter = new AtomicLong(0);
-  private final String postfix;
```

Comment:
backed this change out as we no longer need to post fix the query ids as the ExecutionContexts used by the engines have their own id generators.